controls/SV-204603.rb

```diff
@@ -45,7 +45,7 @@
   tag gtitle: 'SRG-OS-000355-GPOS-00143'
   tag satisfies: ['SRG-OS-000355-GPOS-00143', 'SRG-OS-000356-GPOS-00144']
   tag gid: 'V-204603'
```

Comment:
Need to log an issue to avoid inspec exec error:
```
[2023-05-10T12:08:18-04:00] WARN: Input 'max_poll' does not have a value. Use --input-file or --input to provide a value for 'max_poll' or specify a  value with `input('max_poll', value: 'somevalue', ...)`.
[2023-05-10T12:08:18-04:00] WARN: Input 'max_poll' does not have a value. Use --input-file or --input to provide a value for 'max_poll' or specify a  value with `input('max_poll', value: 'somevalue', ...)`.
```
Although the default value is accessible and it shows in the report, this output can be avoided by assigning the input value to a variable within the control.

Reply:
Logged issue #194